package.json

```diff
@@ -44,7 +44,6 @@
     "sinon": "14.0.0",
     "source-map": "^0.8.0-beta.0",
     "stacktrace-parser": "^0.1.10",
-    "typescript": "^4.6.4",
```

Comment:
I'd be a little hesitant about moving this to devDependencies. It seems innocent and correct, but many of the development-related packages here are in dependencies because the SDK itself does development and compilation. I can't recall specifically if `typescript` is one of the ones that has to remain here, but my intuition is that this is going to break something downstream, so I'd recommend keeping it where it is.

Reply:
Fair enough, will move back.